### PR TITLE
Add filename pattern to cli

### PR DESCRIPTION
**Pull request by Marishka17**
### Motivation and context

Add ability to create task with `filename_pattern` by cli.
### How has this been tested?

Manually
```
cvat-cli --server-port 3000 create "task with filtered cloud storage data" --labels '[{"name": "car"}]' --use_cache --cloud_storage_id 1 --filename_pattern "test_images/*.jpeg" share manifest.jsonl
```
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat-cli/src/cvat_cli/parser.py [hunk 1]
```diff
@@ -201,6 +201,22 @@ def make_cmdline_parser() -> argparse.ArgumentParser:
...
+        help="cloud storage ID if you would like to use data from cloud storage",
+    )
+    task_create_parser.add_argument(
+        "--filename_pattern",
+        type=str,
+        help="""pattern for filtering data from the manifest file for the upload.
+            Only shell-style wildcards are supported:
+            * - matches everything
+            ? - matches any single character
+            [seq] - matches any character in 'seq'
```
Comment (nmanovic) [inline cvat-cli/src/cvat_cli/parser.py:217]: Does it support [abc] and [a-c]?

**Comment by nmanovic** [comment]
@Marishka17 , need to add a line into CHANGELOG.md

**Comment by nmanovic** [comment]
@zhiltsov-max , could you please look?

File: cvat-cli/src/cvat_cli/parser.py (hunk 1 above)
Comment (Marishka17) [reply cvat-cli/src/cvat_cli/parser.py:217]: Yes, both options

File: cvat-cli/src/cvat_cli/parser.py [hunk 2]
```diff
@@ -201,6 +201,22 @@ def make_cmdline_parser() -> argparse.ArgumentParser:
...
+    task_create_parser.add_argument(
+        "--cloud_storage_id",
+        default=None,
+        type=int,
+        help="cloud storage ID if you would like to use data from cloud storage",
+    )
+    task_create_parser.add_argument(
+        "--filename_pattern",
+        type=str,
+        help="""pattern for filtering data from the manifest file for the upload.
```
Comment (zhiltsov-max) [inline cvat-cli/src/cvat_cli/parser.py:213]: Multi-line help is not supported by argparse by default.

```
 --filename_pattern FILENAME_PATTERN
                        pattern for filtering data from the manifest file for the upload. Only shell-style wildcards are supported: * - matches
                        everything ? - matches any single character [seq] - matches any character in 'seq' [!seq] - matches any character not in seq
```

Comment (Marishka17) [reply cvat-cli/src/cvat_cli/parser.py]:
```
--filename_pattern FILENAME_PATTERN
                        pattern for filtering data from the manifest file for the upload.
                        Only shell-style wildcards are supported:
                        * - matches everything
                        ? - matches any single character
                        [seq] - matches any character in 'seq'
                        [!seq] - matches any character not in seq
```

**Review by zhiltsov-max — APPROVED** [review]

**Pull request merged**